add note about using sqlite3 for local development with sqlite (#1201)
fermyon/developer: content/cloud/noops-sql-db.md

See [preparing an SQLite Database](https://developer.fermyon.com/spin/sqlite-api-guide#preparing-an-sqlite-database) for more information on using the `spin up` subcommand's `--sqlite` option to create and populate tables locally. No **Cloud** database is created or changed as a result of using this this optional `spin up --sqlite` example. Populating this local database can be useful for testing (the relationship between your application's logic and your schema) before deploying your application to Fermyon Cloud.

> When developing locally, you may also want to test queries, check tables were created properly, or inspect the data in your local SQLite database. [`sqlite3`](https://sqlite.org/cli.html) is a command line program that allows users to evaluate queries interactively. Simply use the `sqlite3` CLI and point it to the `.spin/sqlite_db.db` file located in your application directory. _Note: `.spin/sqlite_db.db` is created on `spin up`._

Reviewer: This is subjective, and I am not sure if our house style disagrees with me, but "simply" is one of my pet peeve.  The reader will decide if it's simple or not - if they don't find a thing simple, then telling them it's simple makes them feel stupid for not getting it.  (I feel this way a _lot_, especially when I open cookery books.)  But maybe that's just me.

```suggestion
When developing locally, you may also want to test queries, check tables were created properly, or inspect the data in your local SQLite database. [`sqlite3`](https://sqlite.org/cli.html) is a command line program that allows users to evaluate queries interactively. You can point the `sqlite3` CLI to the `.spin/sqlite_db.db` file, located in your application directory, to view or modify your database directly. _Note: `.spin/sqlite_db.db` is created on `spin up`._
```

Author: That's such a good point. Hadn't thought about it before but I so agree.
